### DISCUSSION TITLE
Indication for Successful Topaz Train

### DISCUSSION THREAD

**Discussion by joonp9** [tbepler/topaz#129] [cryoSPARC]
When using Topaz Train, what is a good value to observe in "Training Precision over Epochs" and "Test Average-Precision over Epochs"? If one or both values are not ideal, what are the recommended parameters to change? 

I picked particles on subset of micrographs using Blob Picker. After iterative 2D classifications, I perform Topaz Train. I have changed expected number of particles, learning rate, radius, etc. However, the value for "Test Average-Precision over Epochs" stays around 0.5 at best. According to cryoSPARC Topaz tutorial, this value should be over 0.7. Is this somewhat absolute?

Thank you,

Joon

**Comment by joonp9** [comment]
Additionally, if the training precision reaches 0.95, does that mean it's a good model even if the test average precision is around 0.3-0.5?

**Reply by Guillawme** [reply]
Some of this was discussed here: https://github.com/tbepler/topaz/discussions/78

**Reply by joonp9** [reply]
Thank you for directing me to very helpful thread!
